dm/syncer/metrics/metrics.go

```diff
@@ -30,302 +31,350 @@ const (
 	BinlogEventCostStageGenQuery      = "gen-query"
 )
 
-// below variables are exported to syncer package.
-var (
-	BinlogReadDurationHistogram = metricsproxy.NewHistogramVec(
+// Metrics groups syncer's metric variables.
+type Metrics struct {
```

Comment:
how about renaming to `TaskMetrics`

Reply:
some metrics are not a property of a single task, for example BinlogMasterPosGauge. Welcome to give a better name 😂

Reply:
In this case I do not have a good name too, the name is too difficult 😭